### PR TITLE
Add preamble to top of all twtxt feeds

### PR DESCRIPTION
**Pull request by prologic**
Example:

Screen Shot 2021-02-12 at 08 43 01

### CODE REVIEW

File: internal/twtxt_handlers.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+		if _, err = w.Write([]byte(preamble)); err != nil {
+			log.WithError(err).Warn("error writing twtxt preamble")
+		}
+		http.ServeContent(w, r, filepath.Base(fn), fileInfo.ModTime(), f)
+		/*
+			if _, err = io.Copy(w, f); err != nil {
+				log.WithError(err).Error("error writing twtxt file")
+			}
+		*/
```
Comment (JonLundy) [inline internal/twtxt_handlers.go:175]: Should this be removed?

**Review by JonLundy — APPROVED** [review]

File: internal/twtxt_handlers.go (hunk 1 above)
Comment (prologic) [reply internal/twtxt_handlers.go:175]: Yes 😁

**Review by aaadonai — APPROVED** [review]

**Comment by prologic** [comment]
FYI I'm reverting this commit as no local users are showing up on twtxt.net's cache at all. cc @JonLundy

**Comment by JonLundy** [comment]
fixed in #391

**Pull request merged**